docs/protocol-design/attack-vectors.md

```diff
@@ -45,6 +47,9 @@ Nano has a number of mechanisms built in to protect from a range of possible att
+  
+| [Network Attacks P2](https://github.com/georgehara/nano/wiki/unofficial) | Additional deep-dive into potential attack vectors & mitigations |
```

Comment:
Was this intended to be a table entry? It's not getting formatted

Reply:
Adjusted into it's own section